index.html

```diff
@@ -272,6 +272,10 @@ <h4 style="margin-left: 20px;">Staff Members</h4>
               <p>"Awesome plugin! I was able to make a SCOREBOARD and a LEADERBOARD on my website using this and some PHP!" ~ Dylan C</p>
             </div>
           </div>
+          <div>
+            <h2>This is a advertisement</h2>
+            <iframe data-aa='563915' src='//ad.a-ads.com/563915?size=728x90' scrolling='no' style='width:728px; height:90px; border:0px; padding:0;overflow:hidden' allowtransparency='true'></iframe>
```

Comment:
1. // means it will do https:// if the browser likes https:// but http if the browser freaks out. 
2. This is why I shouldn't do HTML. 😆